speedRead/public/sketch.js

```diff
@@ -0,0 +1,174 @@
+let socket;
+let lines = [], //parag 
+	carry = [], //words
+	tLifts = [],
+	shouldScroll = true,
+	record,
+	chunk = 4,
+	speed = 5.17;//has to be mapped across maybe 1-100
```

Comment:
If `speed` isnt going to be reassigned anymore use `const` keyword to declare it as opposed to `let`.